doc/advanced-configuration/outgoing-connections.md

```diff
@@ -450,3 +451,39 @@ Run the following function in the MongooseIM shell to verify that the connection
 ```
 
 Note that the output might differ based on your ElasticSearch cluster configuration.
+
+## RabbitMQ connection setup
+
+RabbitMQ backend for [`mod_event_pusher`](../modules/mod_event_pusher.md)
+requires a `rabbit` pool defined in the `outgoing_pools` option.
+They can be defined as follows:
+
+```erlang
+{ougtoing_pools, [
+ {rabbit, host, Tag, WorkersOptions, ConnectionOptions}
```

Comment:
I think it's worth mentioning that this will start as many rabbit pools as many XMPP hosts are served by MongooseIM.